test/base/LoanTermsConsensusProcessRequestTest.js

```diff
@@ -138,12 +138,13 @@ contract('LoanTermsConsensusProcessRequestTest', function (accounts) {
             );
             
             await instance.initialize(loansContract, settings.address, markets.address);
+            const txConfig = { from: loansContract };
 
-            await instance.addSigner(nodeOne)
-            await instance.addSigner(nodeTwo)
-            await instance.addSigner(nodeThree)
-            await instance.addSigner(nodeFour)
-            await instance.addSigner(nodeSix)
+            await instance.addSigner(nodeOne, txConfig);
+            await instance.addSigner(nodeTwo, txConfig);
```

Comment:
why do we need to add the txConfig?

Reply:
Without the txConfig all the tests were failing with a VM Exception error - reason given - Ownable: caller is not the owner

Currently, with the txConfig, there are 2 passing and 8 failing mostly with a RESPONSE_EXPIRED error